crates/compiler/fmt/src/lib.rs

```diff
@@ -21,6 +21,7 @@ pub struct Buf<'a> {
     beginning_of_line: bool,
     line_indent: u16,
     flags: MigrationFlags,
+    pub arena: &'a Bump,
```

Comment:
You actually don't need this - you can just access `self.text.bump()`

Reply:
I had ownership issue using that....